Feature/Education.feature

```diff
@@ -0,0 +1,20 @@
+﻿Feature: Education
+User wants to add education details into Profile Details
+Acceptance Criteria
+User is able to add the education details
+
+@mytag
+ Scenario: 01 Test Add Education sucessfully
+	Given User enter UniversityName as "Delhi University"
+	And   Select CountryOfUniversity as "India"
+	And   Select Title as "B.Tech"
+	And   Enter Degree as "Computer Science"
+	And   Select YearOfGreaduation as "2012"  
+	When  Clicks on Add button on AddEducation page
+	Then  Education should be added to the  profile. The Alert message "Education has been added" is displayed.
+
+
+@mytag		

```

Comment:
Just a minor thing. Not so meaningful to have all tags with the same name. The purpose of it is to make them distinct. Now you cant differentiate them because all are different. Or you could just delete it.